.github/workflows/ci.yml

```diff
@@ -57,14 +56,17 @@ jobs:
           cd git-*
           make prefix=/usr/local all
           make prefix=/usr/local install
+        if: ${{ matrix.otp_vsn != 'latest' }}
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - shell: bash
-        run: |
-          # run linter and docs if latest otp release
-          [[ "${{ matrix.otp_vsn }}" != "${{ env.LATEST_OTP_RELEASE }}" ]] || make elvis docs
       - shell: bash
         run: make
+        if: ${{ matrix.otp_vsn != 'latest' }}
```

Comment:
I think it's ok to run `make` and `make test` on `latest`, even though at times it might be redundant. But at times, erlang will publish newer versions than what we're testing against, and then it's good to build and run the tests on that version too.

Vague memory that **maybe** there are some side-effects or running `make elvis docs` first. Maybe not in functionality, but speed. **Maybe!**

Reply:
Squashed the removal of some conditionals with existing code and force-pushed...